framework/include/problems/FEProblemBase.h

```diff
@@ -134,6 +135,19 @@ enum class MooseLinearConvergenceReason
   DIVERGED_PCSETUP_FAILED = -11
 };
 
+enum class MooseCurrentErrand
```

Comment:
What about `MooseCurrentLoop` or `MooseCurrentComputation`?

Reply:
Yeah, both of those are much better. I'm leaning towards Loop.